packages/kbn-shared-ux-components/src/toolbar/buttons/add_from_library/add_from_library.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { mount as enzymeMount, ReactWrapper } from 'enzyme';
+import React from 'react';
+import {
+  SharedUxServicesProvider,
+  SharedUxServices,
+  mockServicesFactory,
+} from '@kbn/shared-ux-services';
+import { ToolbarButton } from '../primary/primary';
+
+describe('<ToolbarButton />', () => {
+  let services: SharedUxServices;
+  let mount: (element: JSX.Element) => ReactWrapper;
+
+  beforeEach(() => {
+    services = mockServicesFactory();
+    mount = (element: JSX.Element) =>
+      enzymeMount(<SharedUxServicesProvider {...services}>{element}</SharedUxServicesProvider>);
+  });
+
+  test('is rendered', () => {
+    const component = mount(<ToolbarButton iconType="folderOpen" label="Add from library" />);
```

Comment:
This should be testing `AddFromLibrary` component, not `ToolbarButton`. Please extract `ToolbarButton` into its separate test.

Reply:
thank you!! [19e27a5](https://github.com/elastic/kibana/pull/127605/commits/19e27a5f9ace636f58be359a19b05cd0e18b25d7)[19e27a5](https://github.com/elastic/kibana/pull/127605/commits/19e27a5f9ace636f58be359a19b05cd0e18b25d7)